### PR TITLE
Add some tested games

### PR DESCRIPTION
**Pull request by Samsagax**
~~Silly and small~~ games that work with no additional tweaks

### CODE REVIEW

File: tweaks/steam.yaml [hunk 1]
```diff
@@ -87,6 +87,9 @@
 "1368440": # Armor Of Heroes
   compat_tool: proton_513
 
+"727020": # Arcade Moonlander
+compat_tool: proton_63
```
Comment (alkazar) [inline tweaks/steam.yaml:91]: Here and elsewhere: missing indentation

**Comment by alkazar** [comment]
Also, there is now a conflict due to having merged a different PR. Sorry!

**Comment by Samsagax** [comment]
No probs. I'll refactor

**Comment by Samsagax** [comment]
Rebased and hopefully fixed

**Comment by alkazar** [comment]
Great, thanks!

**Pull request merged**